tests/test_validator.py

```diff
@@ -147,7 +151,102 @@ def test_valid_xml_against_wrong_schema(self):
         # Exit correctly with code 0
         self.assertEqual(result.exit_code, 0)
         # The correct output is given
-        self.assertEqual(xml + " is invalid.\n\n", result.output)
+        self.assertEqual("The XML file: SUBMISSION.xml\nis invalid.\n\n",
+                         result.output)
+
+    def test_valid_xml_from_url(self):
+        """Test validating XML from URL."""
+
+        # XML from European Nucleotide Archive browser
+        xml_url = "https://www.ebi.ac.uk/ena/browser/api/xml/SAMEA2620084"
```

Comment:
any requests to external website should be mocked using e.g. https://github.com/getsentry/responses do not call the actual website as it will trigger unpredictable behaviour in the tests if the website is down or network connection is slow.

Reply:
Ah right, no wonder Travis is not approving urls in the tests.